Application/Published/Upgrade-AdminApp/Deploy-EdFiOdsAdminApp.ps1

```diff
@@ -0,0 +1,220 @@
+    Deploys Admin App appliacation to an existing Azure resource group
+.DESCRIPTION
+	Deploys Admin App appliacation to an existing Azure resource group
+.PARAMETER ResourceGroupName
+	Existing resource group name
+.PARAMETER ResourceGroupLocation
+    Existing resource group location
+.PARAMETER AdminAppName
+    A friendly name to help identify AdminApp within Azure.  Must be no more than 64 characters long
+.PARAMETER AppInsightLocation
+	Existing app insight location, mostly same as resouce group location
+.PARAMETER ProductionApiUrl
+	Existing ODS/ API Url that Admin App can connect
+.PARAMETER EncryptionKey
+	Base64-encoded 256 bit key appropriate for use with AES encryption
+.PARAMETER TemplateFileDirectory
+	Points the script to the directory that holds the Ed-Fi ODS install templates.  By default that directory is the same as the one that contains this script.
+.PARAMETER AdminAppVersion
+	Admin app version to be deployed
+.PARAMETER Edition
+	Edition (Test, Release) of the Azure Deploy scripts to deploy.  Defaults to Release.
+.EXAMPLE
+	.\Deploy-EdFiOds.ps1 -ResourceGroupName "Ed-Fi-Ods-Resourcegroup" -ResourceGroupLocation "South Central US" -AdminAppName "AdminApp-Latest" -AppInsightLocation "South Central US" -ProductionApiUrl "https://edfiodsapiwebsite-production-yuw8iui32.azurewebsites.net"
+    Deploys the provided version of the AdminApp to the South Central US Azure region with the default instance name
+#>
+Param(
+
+	[string] 
+	[Parameter(Mandatory=$true)] 
+	$ResourceGroupName,
+
+	[ValidateSet("East US", "South Central US")]
+	[string] 
+	[Parameter(Mandatory=$true)]	
+	$ResourceGroupLocation,	
+
+	[string] 
+	[Parameter(Mandatory=$true)] 
+	$AdminAppName,
+
+	[ValidateSet("East US", "South Central US")]
+	[string] 
+	[Parameter(Mandatory=$true)]	
+	$AppInsightLocation,
+	
+	[string]
+	[Parameter(Mandatory=$true)]
+	$ProductionApiUrl,
+
+	[string]
+	[Parameter(Mandatory=$false)] 
+	$EncryptionKey,
+
+	[string] 
+	[Parameter(Mandatory=$false)] 
+	$TemplateFileDirectory = '$PSScriptRoot',
+
+	[string]
+	[Parameter(Mandatory=$false)]
+	$AdminAppVersion = '2.2.0',
+	
+	[string]
+	[ValidateSet('Release','Test')]
+	$Edition = 'Release'	
+)
+
+Import-Module $PSScriptRoot\..\Dependencies.psm1 -Force -DisableNameChecking
+Import-Module $PSScriptRoot\..\EdFiOdsDeploy.psm1 -Force -DisableNameChecking
+
+Use-Module "AzureRM" "4.3.1"
+Use-Module "AzureRM.profile" "3.3.1"
+
+# Powershell doesn't support using $PSScriptRoot in parameter defaults,
+# so using that value as a marker to update with the real PSScriptRoot value
+if ($TemplateFileDirectory -eq '$PSScriptRoot')
+{
+	$TemplateFileDirectory = $PSScriptRoot;
+}
+
+$CacheTimeOut = "10"
+
+$AdminAppTemplateFile = "$TemplateFileDirectory\OdsAdminApp.Upgrade.json"
+$AdminAppTemplateParametersFile = "$TemplateFileDirectory\OdsAdminApp.parameters.Upgrade.json"
+
+function Get-SqlServerInfo()
+{	
+	$hostName = Read-Host -Prompt "SQL Server Hostname (ex: sql.mydomain.com,1433)"
```

Comment:
Why not provide the host name and port as function parameters? If continuing with the user prompt, would it make sense to specify "SQL Server Hostname and Port Number (ex: sql.mydomain.com,1433)"

Reply:
@stephenfuqua 
I tested with just the hostname. It is working fine. Will be removing the port name. Since the default port is used down the line.